docs/Makefile

```diff
@@ -10,8 +10,9 @@ SOURCEDIR     = .
 BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
-autosummary_clean:
+clean:
 	rm -rfv "$(SOURCEDIR)/_autosummary"
+	rm -rfv "$(SOURCEDIR)/_build"
```

Comment:
Why not
```suggestion
	@$(SPHINXBUILD) -M clean "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```
?

Reply:
Also, the `clean` target should be after the `help` one to keep `make` being equivalent to `make help`.